doc/index.md

```diff
@@ -0,0 +1,5 @@
+# image-builder-cli
+
+The `image-builder-cli` project provides a local entrypoint into the [Image Builder](https://osbuild.org/) ecosystem. It allows you to build various predefined images with [customizations](./01-usage.md#blueprints).
```

Comment:
"a local entrypoint into the Image Builder ecossytem" is absolutely correct, but very hard to understand.

This is basically the first paragraph in the documentation, so it needs to be extremely easy to understand. I think we can borrow the description from cobra:

> Image-builder builds operating system images for a range of predefined
operating systems like Fedora, CentOS and RHEL with easy customizations support.

If you want to highlight the osbuild backend, I suggest putting it into a second sentence.